job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -10,27 +10,30 @@ def AGENT_LABEL="kie-rhel && kie-mem512m"
 
 def NEXT_PRODUCT_VERSION=Constants.NEXT_PROD_VERSION
 def NEXT_PRODUCT_BRANCH=Constants.BRANCH
+def NEXT_PRODUCT_CONFIG_BRANCH=NEXT_PRODUCT_BRANCH
 def NEXT_RHBA_VERSION_PREFIX=Constants.RHBA_VERSION_PREFIX
 
 def CURRENT_PRODUCT_VERSION=Constants.CURRENT_PROD_VERSION
 def CURRENT_PRODUCT_BRANCH='7.59.x'
+def CURRENT_PRODUCT_CONFIG_BRANCH= CURRENT_PRODUCT_BRANCH
 def CURRENT_RHBA_VERSION_PREFIX='7.59.1.redhat-'
 
-def KOGITO_NEXT_PRODUCT_VERSION=NEXT_PRODUCT_VERSION
-def KOGITO_NEXT_PRODUCT_BRANCH=NEXT_PRODUCT_BRANCH
+def KOGITO_NEXT_PRODUCT_VERSION='1.13.0'
+def KOGITO_NEXT_PRODUCT_BRANCH='1.13.x'
+def KOGITO_NEXT_PRODUCT_CONFIG_BRANCH="kogito/${KOGITO_NEXT_PRODUCT_BRANCH}"
 
 def KOGITO_CURRENT_PRODUCT_VERSION='1.11.0'
 def KOGITO_CURRENT_PRODUCT_BRANCH='1.11.x'
+def KOGITO_CURRENT_PRODUCT_CONFIG_BRANCH="kogito/${KOGITO_CURRENT_PRODUCT_BRANCH}" // or CURRENT_PRODUCT_BRANCH?
```

Comment:
I don't see the `kogito/1.11.x` branch in the build-configurations repository.

Reply:
Thanks, checked the original version and changed it to CURRENT_PRODUCT_BRANCH.

Reply:
In fact, I have not used the CURRENT_PRODUCT_BRANCH constant (rather the value), since it might be misleading. The value will change with the next update so it could be a source of mistakes.